packages/apify/src/configuration.ts

```diff
@@ -148,6 +149,7 @@ export class Configuration extends CoreConfiguration {
         ACTOR_INPUT_KEY: 'inputKey',
         ACTOR_MEMORY_MBYTES: 'memoryMbytes',
         ACTOR_RUN_ID: 'actorRunId',
+        [ACTOR_ENV_VARS.STANDBY_PORT]: 'standbyPort',
```

Comment:
I would rather do this to be consistent

```suggestion
        ACTOR_STANDBY_PORT: 'standbyPort',
```